server/githookHandler.go

```diff
@@ -0,0 +1,84 @@
+package server
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/eventgrid/2018-01-01/eventgrid"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/date"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/buffalo/render"
+	"github.com/google/go-github/github"
+	"github.com/google/uuid"
```

Comment:
There are a thousand and one UUID libraries in Go. There's probably no difference, but I'd hate to add yet-another UUID implementation to the set of code-bases we use. I'd swap this out for one of the following (prioritized as best first):
1. [`github.com/satori.go/uuid`](https://github.com/satori/go.uuid)
1. [`github.com/Azure/azure-amqp-common-go/uuid`](https://github.com/Azure/azure-amqp-common-go/uuid)
1. [`github.com/marstr/guid`](https://github.com/marstr/guid)